include/deal.II/distributed/tria.h

```diff
@@ -1217,6 +1217,16 @@ namespace parallel
       std::vector<bool>
       mark_locally_active_vertices_on_level(const int level) const;
 
+      virtual unsigned int
+      translate_coarse_cell_id_to_coarse_cell_index(
+        const unsigned int coarse_cell_id) const override;
```

Comment:
Didn't you rename this in some other PR to drop the `translate_` prefix? I actually like the prefix, but I think @tjhei didn't.

You'll also want to use the new data type for coarse cell ids here.

Reply:
@bangerth  I also prefer the `translate_` prefix because it indicates that it is a method. However, I have made this PR now consistent with the previous PRs.

Reply:
Let's go with this then. I wouldn't mind if you had a follow-up PR that changes the name back again. It should be easy to do with find-replace.